modules/ROOT/pages/_partials/sync/introducing-data-sync.adoc

```diff
@@ -0,0 +1,46 @@
+// The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
+[id="introducing-data-sync-{context}"]
+
+// The `context` attribute enables module reuse. Every module's ID includes {context}, which ensures that the module has a unique ID even if it is reused multiple times in a guide.
+= Introducing Data Sync
+
+Data Sync is a JavaScript framework that enables a developer to add real time data synchronization to both mobile and web clients.
+The Data Sync framework also provides offline capabilities that allow a client to continue operating offline and once connectivity is re-established, the client is automatically synchronized.
+An app built using the Data Sync framework typically connects to a data source for data persistence however, an app built using the Data Sync framework works without a data source.
+
+An app built using the Data Sync framework comprises of two components:
+
+* The Sync Client is a JavaScript client offering client side extensions and server side integration. The Sync Client can be integrated into frameworks such as React and Angular.
+
+// tag::excludeDownstream[]
```

Comment:
remove diagrams

Reply:
Completed as part of https://github.com/aerogear/mobile-docs/pull/691/commits/6a273828dc7f778021af3fb6ff9f967f9003c58e